### PR TITLE
Add 'Meet Capact maintainers (..)' blog post

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Add **'Meet Capact maintainers and learn how to become one!'** blog post

Website preview: https://234b5f66.website-8qa.pages.dev/blog/maintainers-interview

Co-authored-by: Paweł Kosiec <pawel@capact.io>

### CODE REVIEW

**Review by mkuziemko — APPROVED** [review]
I read carefully and I didn't spot mistakes. Very good work! :1st_place_medal:

**Review by tmitchell — CHANGES_REQUESTED** [review]
I personally enjoyed reading this to get to hear more of your backstory!

File: blog/2021-12-13-maintainers-interview.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+  - dry
+  - interview
+  - maintainers
+  - contributors
+  - career
+---
+
+Did you know that it's already been 16 months since the Capact was born?
+
+In the previous blog post, you learned which problems Capact aims to solve. This time, Capact maintainers took some time to answer questions you might have if you're considering contributing to the project.
```
Comment (sunny-dasgupta-braingu) [inline blog/2021-12-13-maintainers-interview.md:23]: In a previous blog post, we discussed some of the problems Capact aims to solve. This time, Capact maintainers took some time to answer questions you might have if you're considering contributing to the project.

File: blog/2021-12-13-maintainers-interview.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+What's it like to build cloud-native platform from scratch? What is it like to work in Capact? What are the biggest challenges? How to start with Go? Read on to get the answers!
+
+<!--truncate-->
+
+## Tell us why you are here. How did you begin your adventure in IT?
+
+**Mateusz:** In the past, I worked on similar projects that aimed to unify our developer tooling. In my opinion, it is something that awaits us in the future. It’s definitely worth being a part of it.
+
+My journey with IT actually started thanks to my brother. The first program that I wrote was on Commodore 64 when I was about 10 years old. Since then, I have been involved in technology. From the beginning, I felt love for backends, but tolerated frontend 😁 During my studies, I got interested in the cloud. As a result, I started an internship, where from Java I switched to Go. Now it’s been 5.5 years that I’ve been connected with the cloud-native world and I certainly don’t regret that.
```
[the remark this answers is not in the document]
Comment (sunny-dasgupta-braingu) [reply blog/2021-12-13-maintainers-interview.md:33]: Agreed. If you're going to mention your brother, a quick one sentence on how your brother got you started would be good.

File: blog/2021-12-13-maintainers-interview.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+
+Go, on the other hand, is a great fit for such projects—its simplicity enables us to write such back-end services efficiently. They are fast to compile and consume much fewer resources than alternatives written in Java or JavaScript/TypeScript.
+
+**Mateusz:** It’s worth highlighting that the Go community already created a lot of powerful libraries. You don’t need to solve cloud-specific problems once again, and can focus on solving your domain/business problems instead.
+
+As mentioned, Go is easy in terms of memory consumption, which is important in cloud environments. Of course, it doesn't come without a downside; the abstraction level in Go is quite low, but the upcoming Go 1.18 will be a significant release, e.g. support for generic.
+
+## Can you give us more background on what Capact is?
+
+**Paweł:** Capact is an open-source framework for running arbitrary workflows. Initially, we are focused on day-one and day-two workflows for deploying, and managing applications and cloud-native infrastructure.
```
Comment (sunny-dasgupta-braingu) [inline blog/2021-12-13-maintainers-interview.md:53]: Capact is an open-source framework for running arbitrary workflows. Currently, we are focusing on day-one and day-two workflows for deploying, and managing applications and cloud-native infrastructure.

**Review by sunny-dasgupta-braingu — CHANGES_REQUESTED** [review]

File: blog/2021-12-13-maintainers-interview.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+
+**Paweł:** Apart from balancing between two scopes, open-source and commercial, these two are also separate challenges. We are about to start building an open-source community around Capact, which will definitely be a challenge to gain adoption—not only from hobbyists but also big tech companies.
+
+There's also a commercial offering planned, which is based on Capact. And that's happening right now, so we need to ensure that the Capact user experience is seamless for every user profile (BTW they are described on [our website](https://capact.io/community/user-profiles)) and that it works well in a production environment.
+
+
+## What kind of benefits related to career development can a contributor gain in this project?
+
+**Paweł:** Capact consists of many technologies, and you can learn a lot. Even if you know some of them, while working on Capact, you will have an opportunity to dive deeply into some stuff you have never tried. Apart from Go and Kubernetes, we use GraphQL, TypeScript, Terraform, and plenty of projects and tools around Kubernetes, such as Argo workflows, and MinIO.
```
[the remark this answers is not in the document]
Comment (pkosiec) [reply blog/2021-12-13-maintainers-interview.md:78]: That's good idea, we will do it 👍

**Review by tmitchell — APPROVED** [review]

**Pull request merged**